packages/redux/src/products/serverInitialState/lists.ts

```diff
@@ -46,7 +46,7 @@ const serverInitialState: ListsServerInitialState = ({
   delete query.json;
 
   const builtSlug = getSlug(pathname);
-  const isSet = model?.pageType === 'sets';
+  const isSet = model?.pageType === 'set';
```

Comment:
Question/suggestion - instead of a hardcoded "set" value, what about getting it from the TS enum of the model type? I mean something like `Model['pageType'].Set` (not sure if it's correct though)

Reply:
I don't really know 🤔 

I'll try to understand it and get back to you, in the meantime I'm resolving this, no need to block because of it.